src/Utilities/ProjectData.h

```diff
@@ -113,6 +122,8 @@ class ProjectData : public OhmmsElementBase
 
   ///max cpu seconds
   int max_cpu_secs_;
+
+  std::string driver_epoch_;
```

Comment:
string is bad. Can we use enum? When being passed in via the constructor, a string make me think it is for m_title.

Reply:
To deal with this well I have to make InputSection which can handle this kind on input enum able to deal with the application parameter which has many attributes.  I'd have more changes to InputSection coming in EstimatorManagerInput PR's and would prefer not to include it in this one.

Reply:
We don't need to and maybe should not pull InputSection everywhere. In this use case, just compare strings after reading from XML and keep the enum around should be good enough.